### DISCUSSION TITLE
DY pT binned sample validation plot

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#37] [Comments_HIG_23_001]
Add the DY pT binned sample validation. @jialin-guo1

**Comment by jialin-guo1** [comment]
<img width="241" alt="image" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/1c064cef-7e77-4359-868d-a59c1db0148f">

<img width="247" alt="image" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/b2913cbe-74f0-4b7f-8996-512e8e92d1c3">

**Comment by jialin-guo1** [comment]
[DY_pt_spectrum.pdf](https://github.com/ram1123/2l2Q_limitSettingTool/files/12321470/DY_pt_spectrum.pdf)


this plot only apply lepton selection(saved as .pdf)

**Reply by jialin-guo1** [reply]
I update this plot to pt_2l

**Reply by ram1123** [reply]
What is the selection?

**Reply by ram1123** [reply]
BTW, this plot looks awesome. :)

**Reply by jialin-guo1** [reply]
I only applied leptonic Z selection on this plot

**Comment by ram1123** [comment]
okay. Thanks.

------------------------------------------------------
*Ram Krishna Sharma*
Institute of High Energy Physics
Beijing, China
Skype-ID: *ram1123*
Phone - +41-778159387
------------------------------------------------------
